lib/AST/Decl.cpp

```diff
@@ -5599,6 +5599,53 @@ bool VarDecl::isPropertyMemberwiseInitializedWithWrappedType() const {
       this, /*checkDefaultInit=*/false);
 }
 
+static bool isInitFirstParamEscapingClosure(const ConstructorDecl *init) {
+  Type initTy = init->getInterfaceType();
+  if (!initTy || !initTy->is<AnyFunctionType>())
```

Comment:
Better to assume a type has been set here. It would be weird if it wasn't because then the result of the caller would depend on type checking order.

Reply:
Okay. Removed the check in 1bc9dbd.